kitsune/gallery/migrations/0006_change_bn_BD_and_bn_IN_to_bn.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-08-16 17:54
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def change_locale_bn_bd_and_bn_in_to_bn_forwards(apps, schema_editor):
+    Image = apps.get_model('gallery', 'Image')
+    Video = apps.get_model('gallery', 'Video')
+
+    Image.objects.all().filter(locale='bn-BD').update(locale='bn')
+    # Change bn-IN images to bn
+    Image.objects.all().filter(locale='bn-IN').update(locale='bn')
+
+    Video.objects.all().filter(locale='bn-BD').update(locale='bn')
+    # Change bn-IN videos to bn
+    Video.objects.all().filter(locale='bn-IN').update(locale='bn')
+
+
+def change_locale_bn_to_bn_bd_backwards(apps, schema_editor):
```

Comment:
I would go with just `pass`. This is a one way migration and we cannot preserve the initial data here. So instead of moving everything back to `bn-BD` let's keep them in `bn`.

Reply:
Let's update the backwards migration to `pass` for all the data migrations that we have.